SurfaceToolbox/SurfaceToolbox.py

```diff
@@ -387,7 +387,8 @@ def installRemeshPrerequisites(force=False):
           import pyacvd
       except ModuleNotFoundError as e:
           if force or slicer.util.confirmOkCancelDisplay("This function requires 'pyacvd' Python package. Click OK to install it now."):
-              slicer.util.pip_install("pyacvd")
+              # forcing this version is needed according to https://discourse.slicer.org/t/surface-toolbox-of-3d-slicer-do-not-work-in-windows/40717/4
+              slicer.util.pip_install("pyacvd==0.3.1")

```

Comment:
@lassoan hardcoding a version here is probably best instead of leaving it open ended because this extension is bundled into Slicer core per https://github.com/Slicer/Slicer/blob/07bd0ab99307db86a54748bf092de87ce07c8ac8/SuperBuild.cmake#L339 and we would want stable Slicer uses to have a consistent experience post release.

Reply:
Not entirely clear to me why this extension is bundled into Slicer core versus installed as an extension which support updates post Slicer release. Seems like some other extensions could be bundled into Slicer while others removed to only be extensions.

Reply:
There are a number of these "extensions" that are bundled with Slicer: MultiVolumeExplorer, MultiVolumeImporter, SimpleFilters, BRAINSTools, CompareVolumes, LandmarkRegistration, SurfaceToolbox. I think overall it is a valid idea that the factory-built Slicer can contain bundled extensions. However, I don't really like that they are somewhat irregular extensions (and the bundling mechanism is not exactly the same as for custom applications), and the choice of modules that are brought in like this seems quite arbitrary (probably it is because these extensions were not maintained by the Slicer core developers). So, it may make sense to revise them. I would like to retire MultiVolumeExplorer, MultiVolumeImporter extensions. CompareVolumes, LandmarkRegistration, maybe even SurfaceToolbox could move into the Slicer core repository.